src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/01-hosts.e2e-spec.ts

```diff
@@ -0,0 +1,64 @@
+import { HostsPageHelper } from '../cluster/hosts.po';
+
+describe('Hosts page', () => {
+  const hosts = new HostsPageHelper();
+
+  before(function () {
+    if (!Cypress.env('WITH_ORCHESTRATOR')) {
+      this.skip();
+    }
+  });
```

Comment:
I think we can remove this by adding `"**/orchestrator/*"`  to `ignoreTestFiles` in `cypress.json`.
```suggestion
```

Reply:
Fixed. I removed the variable and use the ignore list.